src/lightning/app/cli/commands/cp.py

```diff
@@ -131,7 +131,7 @@ def _upload_files(live, client: LightningClient, local_src: str, remote_dst: str
             )
             upload_urls.append(response)
 
-    upload_urls = [upload_url.get().upload_url for upload_url in upload_urls]
+    upload_urls = [upload_url.get().upload_id for upload_url in upload_urls]
```

Comment:
this wont work with the new API, you'll need to extend the list by `[url.url for url in upload_url.get().urls]` also make a call to to complete the upload correctly